app/models/project.rb

```diff
@@ -29,6 +30,10 @@ class Project < ActiveRecord::Base
     event :reject do
       transitions from: :proposed, to: :rejected, after: -> { self.comments_locked = true }
```

Comment:
Since we have an additional state `pending` now, we need to also allow transitioning from `:pending` in addition to `:proposed`.